e2e/components/DataTable/DataTable-test.avt.e2e.js

```diff
@@ -276,16 +276,19 @@ test.describe('@avt DataTable', () => {
         'components-datatable-selection--default'
       );
     });
-    test('@avt-advanced-states with-radio-expansion', async ({ page }) => {
+    test.skip('@avt-advanced-states with-radio-expansion', async ({ page }) => {
```

Comment:
Had added them because they were failing and we had spoke about these that we gonna skip them for now and then we can deal with these once we get this PR done. I'll look into these and get them fixed if needed now.